NOTES-DJGPP.md

```diff
@@ -32,6 +32,34 @@ Notes for the DOS platform with DJGPP
  running in a DOS box under Windows. If so, just close the BASH
  shell, go back to Windows, and restart BASH. Then run `make` again.
 
+Cross compilation
+=================
```

Comment:
This isn't the title of the whole document, so better to make the style like the RUN-TIME CAVEAT title below

```suggestion
 CROSS COMPILATION
 -----------------

```